### PR TITLE
PFW-1331 Reset and Disable MMU

### PR DESCRIPTION
**Pull request by vintagepc**
- Fix non-functional "Disable MMU" menu choice (I hope)
- Add EEPROM var to handle MMU enable/disable
- Add menu items. 

TODOs: 

- [x] Test. Sim won't get past selector homing...
- [ ] Translation work for "Disable MMU" menu item. 
- [ ] FW update error cannot be handled by current architecture as a one-shot error.

### CODE REVIEW

**Comment by vintagepc** [comment]
Tested as far as I can on my own printer:
- Disable/enable works and survives a power cycle
- Reset does a soft MMU reset
Tested on Sim:
- TMC overheat, which offers "reset MMU" option.
- FW incorrect disable.

File: Firmware/mmu2.cpp [hunk 1]
```diff
@@ -103,7 +103,7 @@ void MMU2::Start() {
 
     mmu2Serial.begin(MMU_BAUD);
 
-    // PowerOn(); we cannot do that on MK3, but at least reset the MMU
+    PowerOn(); // I repurposed this to serve as our EEPROM disable toggle.
```
Comment (DRracer) [inline Firmware/mmu2.cpp:106]: interesting idea :+1:

File: Firmware/mmu2.cpp [hunk 2]
```diff
@@ -549,6 +555,7 @@ void MMU2::manage_response(const bool move_axes, const bool turn_off_nozzle) {
             st_synchronize(); 
             return;
         case VersionMismatch: // this basically means the MMU will be disabled until reconnected
+            CheckUserInput();
```
Comment (DRracer) [inline Firmware/mmu2.cpp:558]: At this moment the only thing we can do is disabling the MMU. In the future we'll be able to flash the new FW as well thanks to @leptun

**Comment by DRracer** [comment]
@vintagepc well done :+1: 
TODOs:
- translation work will be done at some later stage, especially will all the error texts
- let's fix the FW update error later, it will come handy with the generic read/write support for the MMU (which breaks the protocol a bit so that an update of FW will be necessary)

**Pull request merged**